jaxws-retail/service/pom.xml

```diff
@@ -80,6 +86,7 @@
                 <artifactId>jaxws-tools-maven-plugin</artifactId>
                 <version>${version.jaxws-tools-maven-plugin}</version>
                 <configuration>
+                    <fork>true</fork> <!--required when using JDK-9-->
```

Comment:
Do we know the reason for this change in JDK 9?

Reply:
Asking because `fork` usually means workaround. We should know the reasons because it can happen that this won't work on Java 10 or 11.

http://jdk.java.net/10/
http://jdk.java.net/11/

Reply:
Fork in this case is ok. As otherwise you need to start the maven instance with extra --add-modules to make it work.

But in any case, plugin is broken in sense on how it invokes fork, as it just invokes "java" executable which in case that it is not on path, wont work. That is why windows CI job fails.
As result this only works if JDK < 9 is on executable path which successfully invokes the generation.

Or in short. this workaround does *NOT* work.